_posts/2022-07-29-PC-Engines-Firmware-v4-17-0-2.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.17.0.2"
+date:   2022-07-29
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.17.0.2
+
+## Key changes
+
+Mainline:
+
+### Changed
+
+- **Rebased** with official coreboot repository **commit** df721bd
+
+## Statistics
+
+
+![Files Changed](https://cloud.3mdeb.com/index.php/s/mAjYHr4FC2MXW5R/preview)
+
+The chart shows the total files changed from release tag against the rebase
+point of given release specified in CHANGELOG (CHANGELOG.md, gitlab-ci.yml
+and regression.sh excluded from statistics). Check the statistics with:
+
+```
+git diff --stat df721bd ':(exclude).gitlab-ci.yml' ':(exclude)CHANGELOG.md' ':(exclude).gitlab-ci/regression.sh'
+```
+
+`117 files changed, 4246 insertions(+), 219 deletions(-)`
+
+![Process of mainlining](https://cloud.3mdeb.com/index.php/s/xxtmnAzGTWMHKpx/preview)
+
+The chart represents the total line added and deleted on the PC Engines
+coreboot fork against the rebase point for a given release.
+
+## Testing
+
+* [PC Engines hardware configuration matrix](https://cloud.3mdeb.com/index.php/s/GYecq2SHidoFZ8A/preview) -
+  hardware configurations available for testing in 3mdeb laboratory.
+
+* [PC Engines release validation results](https://docs.google.com/spreadsheets/d/1_uRhVo9eYeZONnelymonYp444zYHT_Q_qmJEJ8_XqJc/edit#gid=2507399) -
+  please note there are separate sheets for each board-release.
+
+![Mainline test results](https://cloud.3mdeb.com/index.php/s/6sE8cFpCZ86RGAH)
+
+* Mainline:
+  * PASSED: 763 (**+9**)
+  * FAILED: 10 (**-2**)
+  * PASSED [%]: 98,58 (**+0,15**)
```

Comment:
@Stojek139808 and passes are related to what? Please us correct changelog information explaining what was added.

@macpijan @sulewskiprzemyslaw now those newsletters are extremely boring and useless. Please start improving information that you communicate to end users. @BeataZdunczyk we had this discussion and I really have hope state of release notes will change for this project since we can't accept so low quality.

We also discussed power management improvement, I saw some documentation why it not linked ?

Reply:
@pietrushnic documentation describing power consumption has been linked in the newsletter, I can also add this section in this blog post.

Reply:
@pietrushnic is it more clear now?

Reply:
yes, it is more clear, but whole newsletter and blog post should be significantly improved